src/main/java/com/rocketden/main/model/User.java

```diff
@@ -0,0 +1,10 @@
+package com.rocketden.main.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class User {
+    private String nickname;
```

Comment:
Is a nickname the only thing we need to know about a user? Should we also save some other unique identifier like a machine IP address or a randomly generated ID?

Reply:
There is more information we will want in the future; namely, we will include an auto-generated **ID** when the User is placed into the database, and we will associate a **color** with each User in the backend. I didn't include these things in this PR because I wouldn't have used them, and I didn't want to include unused fields / variables.

For now, I don't see any need for storing IP addresses -- putting aside potential legal issues, when we want to identify users we can add accounts rather than tracking them through IP addresses (especially for the MVP, this is not needed).